.github/CODEOWNERS

```diff
@@ -1,5 +1,5 @@
 * @CircleCI-Public/developer-experience
 *orb*.go @CircleCI-Public/orb-publishers @CircleCI-Public/developer-experience
 
-/api/runner @CircleCI-Public/on-prem
```

Comment:
This should stay as @CircleCI-Public/on-prem The runner team was folded into On Prem, and the old team may be removed from github w/o notice.

Reply:
Addressed! FYI no context is given on this PR, just know that it just applies the revert we did Wednesday on the `develop` branch